src/libraries/System.Security.Cryptography.Xml/tests/CanonicalXmlEntityReferenceTests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Text;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public static class CanonicalXmlEntityReferenceTests
+    {
+        [Fact]
+        public static void Write_WritesExpectedOutput()
+        {
+            var xmlDocument = new XmlDocument();
+            var entityReference = new CanonicalXmlEntityReference("entity", xmlDocument, true);
```

Comment:
The tests are failing to compile now because `CanonicalXmlEntityReference` is `internal`. We don't typically unit test internal classes, only the public API surface.